src/objecttypes/setup_configuration/tests/files/token_auth/invalid_setup_empty.yaml

```diff
@@ -0,0 +1,3 @@
+objecttypes_tokens_config_enable: true
```

Comment:
This file seems to be identical to `invalid_setup.yaml`, so I think you can reuse that one wherever this file is used

Reply:
Yes, they are similar, the only thing that differs is the namespace